x/evm/module.go

```diff
@@ -148,7 +148,7 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 
 	// This migration is a Kava specific migration that also includes the
 	// upstream migrations from 3 to 5.
```

Comment:
is this comment still true, or does it only migrate to 3?
or are we running through consensus version 5 migrations on the upstream that only corresponds to our v3?

Reply:
We combined the upstream migrations for consensus versions 3 to 5 into a single migration which is just 3 on our fork. So our version 3 is equivalent to their version 5 (with our additions on top) if that makes sense

Reply:
thanks for the clarification!